operations/agent-flow-mixin/dashboards.libsonnet

```diff
@@ -2,5 +2,7 @@
   grafanaDashboards+:
     (import './dashboards/controller.libsonnet') +
     (import './dashboards/resources.libsonnet') +
-    (import './dashboards/prometheus.remote_write.libsonnet'),
+    (import './dashboards/prometheus.remote_write.libsonnet') +
+    (import './dashboards/cluster-node.libsonnet') +
+    (import './dashboards/cluster-overview.libsonnet'),
```

Comment:
Should these two dashboards be given the deployment annotations like the other dashboards have? The deployment annotation will be especially useful on the clustering dashboards, I feel.

Reply:
Yes, right! They've been added now.